lib/ws_servers/api/handlers/on_save_favourite_trade_pairs.js

```diff
@@ -0,0 +1,35 @@
+const send = require('../../../util/ws/send')
+const sendError = require('../../../util/ws/send_error')
+const validateParams = require('../../../util/ws/validate_params')
+const isAuthorized = require('../../../util/ws/is_authorized')
+const {
+  notifySuccess
+} = require('../../../util/ws/notify')
+
+module.exports = async (server, ws, msg) => {
+  const { d, db } = server
+  const { FavouriteTradePairs } = db
+  const [, authToken, pairs] = msg
+
+  const validRequest = validateParams(ws, {
+    pairs: { type: 'array', v: pairs }
+  })
+
+  if (!validRequest) {
+    return
```

Comment:
should we just abort? maybe showing an error message or at least logging to the console would be nice

Reply:
console.error should be enough